kratos/python/add_vector_to_python.cpp

```diff
@@ -62,14 +62,36 @@ namespace Python
          binder.def("__rdiv__", [](TVectorType vec1, const double scalar){for(unsigned int i=0; i<vec1.size(); ++i) vec1[i]/=scalar;}, is_operator());
         binder.def("__add__", [](const TVectorType& vec1, const TVectorType& vec2){Vector aux(vec1); aux += vec2; return aux;}, is_operator());
         binder.def("__sub__", [](const TVectorType& vec1, const TVectorType& vec2){Vector aux(vec1); aux -= vec2; return aux;}, is_operator());
-         
+
         //access operators
         binder.def("__setitem__", [](TVectorType& self, const unsigned int i, const typename TVectorType::value_type value){self[i] = value;} );
         binder.def("__getitem__", [](const TVectorType& self, const unsigned int i){return self[i];} );
-        
+
+        //access using slices
+        binder.def("__setitem__", [](TVectorType &self, pybind11::slice this_slice, const TVectorType &value) {
+            size_t start, stop, step, slicelength;
+            if (!this_slice.compute(self.size(), &start, &stop, &step, &slicelength))
+                throw pybind11::error_already_set();
+            if (slicelength != value.size())
+                throw std::runtime_error("Left and right hand size of slice assignment have different sizes!");
+            for (size_t i = 0; i < slicelength; ++i) {
+                self[start] = value[i]; start += step;
+            }
+        });
+        binder.def("__getitem__", [](const TVectorType &self, pybind11::slice this_slice) -> TVectorType* {
```

Comment:
@RiccardoRossi Looks like we are returning a raw pointer here. I'm not sure if this is a problem, but I can try returning something else if you think this can cause problems.